app/src/test/java/io/apicurio/registry/noprofile/rest/v2/GroupsResourceTest.java

```diff
@@ -178,6 +178,53 @@ public void testDefaultGroup() throws Exception {
                 .body("openapi", equalTo("3.0.2"))
                 .body("info.title", equalTo("Empty API"));
     }
+    
+    @Test
+    public void testCreateArtifactRule() throws Exception
+    {
+    	String oaiArtifactContent = resourceToString("openapi-empty.json");
+    	createArtifact("testCreateArtifactRule", "testCreateArtifactRule/EmptyAPI/1", ArtifactType.OPENAPI, oaiArtifactContent);
+    	
+    	//Test Rule type null
+    	Rule nullType = new Rule();
+    	nullType.setType(null);
+    	nullType.setConfig("TestConfig");
+    	given()
+	        	.when()
+	    			.pathParam("groupId", "testCreateArtifactRule")
+	    			.pathParam("artifactId", "testCreateArtifactRule/EmptyAPI/1")
+	    			.body(nullType)
+	    			.post("/registry/v2/groups/{groupId}/artifacts/{artifactId/rules}")
```

Comment:
This test is failing due to a missing `}` at the end of artifactId.

Reply:
Thank you @carlesarnal for pointing that out. I have made the corrections now. Please review and let me know for any changes.